lib/resources/kernel_module.rb

```diff
@@ -37,7 +38,7 @@ def loaded?
     end
 
     def version
-      if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
+      if inspec.os[:family] == 'redhat'
```

Comment:
@Fodoj thank you for highlighting this issue. I assume this applies to RedHat 5 as well, therefore we may need to ask for
```
if inspec.os.redhat? && inspec.os[:release].to_i == 5
```